### DISCUSSION TITLE
java client large chunk

### DISCUSSION THREAD

**Discussion by zyjnd** [seaweedfs/seaweedfs#5921] [Q&A]
how can i to chunking large file when uploading by java client

**Comment by chrislusf** [comment]
The client will do it.

**Comment by zyjnd** [comment]
@chrislusf 
"seaweedOutputStream.write"  will do?I want chunking in client

**Comment by zyjnd** [comment]
I found FileChunkManifest.class in java client,can i use it to chunking file by myself?
